### PR TITLE
student profile pic upload: upload button should only be enabled when there is something to upload #4473

### LINKED ISSUE #4473: student profile pic upload: upload button should only be enabled when there is something to upload
The 'upload picture' button should not be enabled if there are no files selected? Another thing: The two buttons look very different.
image

### PR DESCRIPTION
**Pull request by thyageshm**
fixes #4473

### CODE REVIEW

**Comment by thyageshm** [comment]
@ashrayjain please review the pr :-) I will upload the html test fixes soon

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 1]
```diff
@@ -569,6 +579,23 @@ Apply to thead to make headers not bold.
     margin-bottom: 10px;
 }
 
+.profile-pic-file-selector {
+    overflow: hidden;
+    position: relative;
+}
+
+    .profile-pic-file-selector > #studentPhoto {
```
Comment (ashrayjain) [inline src/main/webapp/stylesheets/teammatesCommon.css]: Indentation issue here?

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 2]
```diff
@@ -569,6 +579,23 @@ Apply to thead to make headers not bold.
...
+        position: absolute;
+        top: 0;
+        right: 0;
+        min-width: 100%;
+        min-height: 100%;
```
Comment (ashrayjain) [inline src/main/webapp/stylesheets/teammatesCommon.css]: Indentation issue here?

Comment (thyageshm) [inline src/main/webapp/stylesheets/teammatesCommon.css]: It is intended haha. The indentation is used to specify hierarchical definitions and ease of reading

**Comment by thyageshm** [comment]
hey you told this was ready to merge offline. so I am pushing the branch

**Pull request merged**